### PR TITLE
Add colored HTML, XML attribute names

### PR DESCRIPTION
**Pull request by MartyBeGood**
Added purple coloring for HTML and XML files.

Gets this theme closer to what IntelliJ does to these file types

### CODE REVIEW

File: themes/Darcula.tmTheme [hunk 1]
```diff
@@ -752,6 +751,18 @@ tag.punctuation.end.arrow.parameters.embedded</string>
 				<string>italic</string>
 			</dict>
 		</dict>
+		<dict>
+			<key>name</key>
+			<string>XML, HTML Attributes</string>
+			<key>scope</key>
+			<string>entity.other.attribute-name.xml
```
Comment (rokoroku) [inline themes/Darcula.tmTheme]: missing `,`

**Comment by rokoroku** [comment]
Hi. This is what I've known about Webstorm's style, 
image

And Below is your suggestion.


Do purple coloring for the HTML attributes make this theme closer to what IntelliJ does?

**Comment by MartyBeGood** [comment]
I'm going to have to apologize.
HTML attributes aren't purple in the Jetbrains-IDEs. You are correct.
For some reason, every Darcula-style theme for Eclipse (what I am using at work) renders HTML attributes purple, which led me to believe this was a good idea.
Sorry for taking your time.

**Comment by rokoroku** [comment]
I guess that might be a limitation of eclipse' theme scheme. 
Anyway, nevermind. I appreciate your interest :)

**Pull request closed without merging**